contrib/kubespray/quick-start/services-configuration.yaml.template

```diff
@@ -178,21 +178,25 @@ authentication:
 #       smtp-from: alert-sender@example.com
 #       smtp-auth-username: alert-sender@example.com
 #       smtp-auth-password: password-for-alert-sender
+#       customized-templates: # optional
+#       - your-customized-email-template
```

Comment:
Do we really need this option? Can we parse the folder to get the templates? I think you can modify the deploy python file to achieve this

Reply:
> Do we really need this option? Can we parse the folder to get the templates? I think you can modify the deploy python file to achieve this

Right, but I don't want to touch the python2 code...

Reply:
> Do we really need this option? Can we parse the folder to get the templates? I think you can modify the deploy python file to achieve this

How can we save the parsing results in `cluster_cfg` ?

Reply:
You can change https://github.com/microsoft/pai/blob/df25b985699b75a3813a2080ee6d437ca0db1bf2/src/alert-manager/config/alert_manager.py#L66-L69. Put the parsed result into `result` and return. Then you will get it from `cluster_cfg`

Reply:
> You can change
> 
> https://github.com/microsoft/pai/blob/df25b985699b75a3813a2080ee6d437ca0db1bf2/src/alert-manager/config/alert_manager.py#L66-L69
> 
> . Put the parsed result into `result` and return. Then you will get it from `cluster_cfg`

`alert_manager` is a class definition file, there is no guarantee on the path to pass